calcom/cal.com
# fix: getPublicEventType of NonOrgUser fetch OrgUser event-type of same username

Fix getPublicEventType
- If a non org user has event-types with the same slug than another org user, and if they both have the same username, the wrong event-type is fetched
- Fixed a test that could have caught this bug, if it were correctly implemented - Hariom

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

- create a non org user with username Sean
- create an org user with username Sean
- create event-type for non org user Sean with slug 15min
- create event-type for org user Sean with slug 15min
- access public url of non org user Sean
- the event-type display in the booker should be the one of non org user
